draft-ietf-quic-recovery.md

```diff
@@ -587,7 +587,7 @@ received from the client, because packets sent on PTO count against the
 anti-amplification limit. Note that the server could fail to validate the
 client's address even if 0-RTT is accepted.
 
-Since the server could be blocked until more packets are received from the
+Since the server could be blocked until more bytes are received from the
```

Comment:
In this case, I prefer packets, since that infers that there are bytes received intended for the specific connection.

Reply:
I like datagrams here -- that is the same language in the transport draft.

Reply:
Good point, datagrams is the correct answer I believe.